tasks/SticklerForRules.tex

```diff
@@ -0,0 +1,46 @@
+\section{Stickler for the Rules [Party Host]}
+The robot has to enforce house rules set by the homeowner.
+
+
+\subsection{Main Goal}
+The robot has to recognize when party guests are breaking rules and politely ask them to stop.
+
+\noindent\textbf{Reward:} 500pts (100pts per recognized rule breaking).
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} This test takes place in the arena.
```

Comment:
Location must make clear if the test takes place in the *whole* arena or just in a predefined room